Chapter 1/Solution3.cs

```diff
@@ -0,0 +1,15 @@
+namespace Problem3
+{
+    public class Solution3
+    {
+        public bool GreaterThan5(int a)
+        {
+                if(a>5)
+                {
+                    return true;
+                }
+                else 
+                return false;
```

Comment:
indent the return statement.Also add the paratheses even for single statement.

Reply:
Corrected indentations and added paratheses